hiyapyco/__init__.py

```diff
@@ -185,6 +186,12 @@ def __init__(self, *args, **kwargs):
             self.encoding = kwargs['encoding']
             del kwargs['encoding']
 
+        self.mergeprimitive = lambda a, b, context: b
```

Comment:
In favor of a better readability I would prefer to use here
```diff
diff --git a/hiyapyco/__init__.py b/hiyapyco/__init__.py
index d096b31..6296caf 100644
--- a/hiyapyco/__init__.py
+++ b/hiyapyco/__init__.py
@@ -186,7 +186,7 @@ class HiYaPyCo:
             self.encoding = kwargs['encoding']
             del kwargs['encoding']
 
-        self.mergeprimitive = lambda a, b, context: b
+        self.mergeprimitive = None
         if 'mergeoverride' in kwargs:
             # DEBT: Compare to make sure function signature is right
             self.mergeprimitive = kwargs['mergeoverride']
@@ -427,8 +427,12 @@ class HiYaPyCo:
             logger.debug('pass as b is None')
             pass
         if a is None or isinstance(b, primitiveTypes):
-            logger.debug('deepmerge: replace a "%s"  w/ b "%s"' % (a, b,))
-            a = self.mergeprimitive(a, b, context)
+            if self.mergeprimitive is None:
+                logger.debug('deepmerge: replace a "%s"  w/ b "%s"' % (a, b,))
+                a = b
+            else:
+                logger.debug('deepmerge: call mergeprimitive for a "%s",  b "%s" with context' % (a, b,))
+                a = self.mergeprimitive(a, b, context)
         elif isinstance(a, listTypes):
             if isinstance(b, listTypes):
                 logger.debug(
```

Reply:
Hello @malachib 
you should create a new commit including the change
```diff
-        self.mergeprimitive = lambda a, b, context: b
+        self.mergeprimitive = None
```
or something similar here before resolving it